test/i18next_test.dart

```diff
@@ -198,6 +210,58 @@ void main() {
     });
   });
 
+  group('fallback', (){
```

Comment:
I think you have to do a `flutter format` in this file, otherwise it won't pass CI validation